languages/python/exercises/concept/dicts/.docs/introduction.md

```diff
@@ -0,0 +1,75 @@
+A _dictionary_ is Python's primary mapping type that connects keys with _hashable values_. The looking up of keys is more efficient than searching through an array, but requires more memory. For a detailed explaination of _hashmaps_ visit [this][hashmap-blog] blog post.
```

Comment:
The `introduction.md` should preferrably not link to external resources (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd)). The resource you're linking to though is an excellent one for the `hints.md` and `after.md` files.

```suggestion
A _dictionary_ is Python's primary mapping type that connects keys with _hashable values_. The looking up of keys is more efficient than searching through an array, but requires more memory.
```